src/Elementor/Widgets/CommentList/WidgetCommentList.php

```diff
@@ -2159,7 +2159,7 @@ protected function render() {
     }
```

Comment:
Following code might be better. It is short and simple. But returns NULL if the variable doesn't exist
$commentID = filter_input(  INPUT_POST, 'commentID'  );

Reply:
It´s not only better, with filter_input it will work, with filter_var not :-)
Is fixed